src/main/java/me/dinowernli/grpc/prometheus/Common.java

```diff
@@ -0,0 +1,60 @@
+package me.dinowernli.grpc.prometheus;
+
+import io.grpc.Metadata;
+import io.grpc.Metadata.Key;
+import io.prometheus.client.SimpleCollector;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Encapsulates some common utilities used in both client and server metrics.
+ */
+class Common {
```

Comment:
Having something called "Common" always feels a bit like a smell because that conveys no information to the reader about what they can expect to find in the file/class. Could we call this class "Labels" instead (and update the doc comment)?